### PR TITLE
✨ Use cnspec v9 by default

### PR DESCRIPTION
**Pull request by czunker**

### CODE REVIEW

**Comment by czunker** [comment]
Later, we can remove the flag completely.

**Comment by czunker** [comment]
Tests were initially failing because of https://github.com/mondoohq/cnquery/issues/2297

**Comment by czunker** [comment]
https://github.com/mondoohq/cnquery/pull/2633 should fix the tests.

**Comment by czunker** [comment]
Another fix for container names: https://github.com/mondoohq/cnquery/pull/2719

**Comment by czunker** [comment]
Another issues discovered during the tests: https://github.com/mondoohq/cnquery/issues/2805

**Comment by czunker** [comment]
https://github.com/mondoohq/cnquery/issues/2818 is now also blocking this PR.

File: tests/framework/utils/asset.go [hunk 1]
```diff
@@ -23,6 +23,16 @@ func ExcludeClusterAsset(as []assets.AssetWithScore) []assets.AssetWithScore {
 	return newAssets
 }
 
+func ExcludeNodeAsset(as []assets.AssetWithScore) []assets.AssetWithScore {
+	var newAssets []assets.AssetWithScore
+	for _, asset := range as {
+		if asset.AssetType != "k8s.node" {
```
[the remark this answers is not in the document]
Comment (czunker) [reply tests/framework/utils/asset.go:29]: I cannot find a discovery option for it:
https://github.com/mondoohq/cnquery/blob/main/providers/k8s/resources/discovery.go#L29-L44

I also do not see a node object in the scan logs. I currently do not know where it is coming from. Also, when I scan the cluster from the outside, I do not see a node object.

Any other ideas?

[the remark this answers is not in the document]
Comment (czunker) [reply tests/framework/utils/asset.go:29]: I think, I found it. We have a feature flag for node discovery which is only partially ported to v9. I'll create another PR for cnquery.

[the remark this answers is not in the document]
Comment (czunker) [reply tests/framework/utils/asset.go:29]: https://github.com/mondoohq/cnquery/pull/2878 should fix this.

[the remark this answers is not in the document]
Comment (czunker) [reply tests/framework/utils/asset.go]: Tests are green again without special handling for `k8s-node`.

**Review by imilchev — APPROVED** [review]

**Pull request merged**